src/crypto/tls/common.go

```diff
@@ -1201,6 +1254,7 @@ func (chi *ClientHelloInfo) SupportsCertificate(c *Certificate) error {
 // the server that sent the CertificateRequest. Otherwise, it returns an error
 // describing the reason for the incompatibility.
 func (cri *CertificateRequestInfo) SupportsCertificate(c *Certificate) error {
+	// TODO: should we add a check for DC as well?
```

Comment:
Well?

Reply:
Still thinking about it. It was pushed today. But if people have thoughts, let them known ;)

Reply:
Mmm.. so the dc gets added later and properly checked, so it seems nit needed here.